### ISSUE TITLE
PFX unusable on MacOS

### ISSUE THREAD

**Issue by badfiles** [chris2511/xca#360]
Produced pfx files cannot be applied in MacOS keychain, I get 'wrong password' error
The reason I believe is SHA256 MAC used, SHA1 is expected
The pfx export params should be configurable.
v. 2.4.0

also this is strange
```
openssl pkcs12 -info -in test.pfx 

Enter Import Password:
MAC: sha256, Iteration 2048
MAC length: 32, salt length: 8
PKCS7 Encrypted data: pbeWithSHA1And3-KeyTripleDES-CBC, Iteration 2048                          <--
...
PKCS7 Data
Shrouded Keybag: PBES2, PBKDF2, AES-256-CBC, Iteration 2048, PRF hmacWithSHA256                 <--
Bag Attributes
```

**Comment by legion151** [comment]
Can confirm for android.

**Comment by chris2511** [comment]
Fixed by merging #389
